vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -13,12 +13,12 @@ def textual_group_relationships
   end
```

Comment:
There was an empty method here with a comment block when I commented...it might have been updated as I was commenting